ui/de-webapp/src/main/java/org/iplantc/de/server/controllers/DeController.java

```diff
@@ -46,12 +49,30 @@ public String redirectToDe(final HttpServletRequest request) throws MalformedURL
         return "redirect:de/";
     }
 
+    private DiscoveryEnvironmentMaintenance getDeMaintenance() {
+        return new DiscoveryEnvironmentMaintenance(maintenanceFile);
```

Comment:
In the current code the `DiscoveryEnvironmentMaintenance` is constructed with `System.getProperty("user.dir") + "/" + maintenanceFile`.
Does just passing `maintenanceFile` work here?